OrbitCore/CrashHandler.h

```diff
@@ -3,10 +3,22 @@
 //-----------------------------------
 #pragma once
 
+#include <atomic>
 #include <string>
 
+#ifdef _WIN32
+#include <client/crashpad_client.h>
+#endif
+
 class CrashHandler {
  public:
-  CrashHandler();
-  static void SendMiniDump();
+  explicit CrashHandler(const std::string& dump_path,
+                        const std::string& handler_path);
+  void SendMiniDump();
+
+ private:
+  inline static bool is_init_{false};
```

Comment:
This is the first time I see a static variable declared with "inline".  C++ 17 ftw! Cool.